TLSharp.Core/TelegramClient.cs

```diff
@@ -25,14 +25,24 @@ namespace TLSharp.Core
     public class TelegramClient : IDisposable
     {
         private MtProtoSender sender;
+        private AuthKey _key;
```

Comment:
please rename variable to not have underscore (to be consistent with the rest of the vars)